index.md

```diff
@@ -60,7 +60,7 @@ description: Vanilla OS is an Immutable Linux-based distribution which aims to p
             <h2>Play.</h2>
             <p>Your teammate by night.</p>
             <p>Play your favourite video games without worrying about setting up your devices. Vanilla OS offers the latest stable Linux kernel, ensuring support for the latest devices and the latest performance improvements.</p>
-            <p>Thanks to an integrated driver manager, your GPU will no longer be a problem, be it AMD®, Intel® or NVIDIA®, if something is missing, just open the driver manager and follow the suggestions, it's that simple.</p>
+            <p>Your GPU won't be an issue anymore thanks to an integrated driver management, whether it's an AMD®, Intel®, or NVIDIA® GPU. If a driver is missing, all you have to do is launch the driver manager and follow the instructions. <strong>It's that simple.<strong></p>
```

Comment:
```suggestion
            <p>Your GPU won't be an issue anymore thanks to an integrated driver management, whether it's an AMD®, Intel®, or NVIDIA® GPU. If a driver is missing, all you have to do is launch the driver manager and follow the instructions. <b>It's that simple.</b></p>
```
Yep, as expected in Jekyll I tested the page locally and the div got broken due to the strong tab. We can use Bold `<b>` instead of `<strong>`.

Reply:
Fixed